pallet-staking/src/weights.rs

```diff
@@ -243,6 +244,11 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
             .saturating_add(T::DbWeight::get().reads((1 as Weight).saturating_mul(w as Weight)))
             .saturating_add(T::DbWeight::get().writes(2 as Weight))
     }
+    fn set_slashing_beneficiary() -> Weight {
+        (20_228_000 as Weight)
+            .saturating_add(T::DbWeight::get().reads(2 as Weight))
```

Comment:
Not sure about the weight, I think it's 1 read and 1 write.

read:
```
ensure_root(origin)?;
```

write:
```
SlashingBeneficiary::<T>::set(target);
```